### PR TITLE
Changes documentation style and layout

### PR DESCRIPTION
**Pull request by vinpac**
Hi, I love GraphiQL, but I find its documentation use experience not very good. So, I made some changes to the layout and style to improve the user experience.

## Schema

Image of Yaktocat

## Type

Image of Yaktocat

## Search on Field

Image of Yaktocat

## Field

Image of Yaktocat

### Another example of field:

Image of Yaktocat

### CODE REVIEW

File: src/components/DocExplorer/SearchResults.js [hunk 1]
```diff
@@ -127,7 +127,11 @@ export default class SearchResults extends React.Component {
 
     if (withinType && matchedTypes.length + matchedFields.length > 0) {
       return (
-        <div>
+        <div className="doc-category">
+          {matchedWithin.length ?
+            <div className="doc-category-title">
+              {`Search results on ${withinType.name}`}
```
Comment (asiandrummer) [inline src/components/DocExplorer/SearchResults.js]: nit: `Search Results` => lets uppercase the first letters

**Comment by asiandrummer** [comment]
Looks good to me - thanks for the contribution :D could we make a couple of changes, especially with other keywords (e.g. arguments, other results, etc) to have the first letter in upper case? (Arguments, Other Results, etc)

Also, the argument list (for example below `hasArgs`) seems a bit cluttered - can we allow some room between each argument?

I'm also interested what @wincent and @AGS- think of this design - could you guys weigh in?

File: css/doc-explorer.css [hunk 2]
```diff
@@ -269,3 +320,11 @@
   top: 50%;
   transform: translate(0, -50%);
 }
+
+.graphiql-container .doc-alert-text {
+  border-top: 1px solid #ddd;
+  text-align: center;
+  padding: 15px;
+  color: #666;
+  display: block;
```
Comment (wincent) [inline css/doc-explorer.css]: nit: can we alphabetize props?

File: src/components/DocExplorer.js [hunk 3]
```diff
@@ -124,19 +124,21 @@ export class DocExplorer extends React.Component {
...
-            <SearchBox
-              value={navItem.search}
-              placeholder={`Search ${navItem.name}...`}
-              onSearch={this.handleSearch}
-            />}
+            <div className="search-box-outer">
+              <SearchBox
+                value={navItem.search}
+                placeholder={`Search ${navItem.name}...`}
+                onSearch={this.handleSearch}
```
Comment (wincent) [inline src/components/DocExplorer.js]: alpha

**Comment by wincent** [comment]
> I'm also interested what @wincent and @AGS- think of this design - could you guys weigh in?

Alas, this is a totally subjective call and I am not sure whether any one person's opinion is super valuable here. To my eye, the extra `border` increases visual clutter and is not necessarily a good thing. But I am just a lowly JavaScript engineer... Would be good do get some feedback from someone with actual design skills. @leebyron?

**Comment by vinpac** [comment]
> could we make a couple of changes, especially with other keywords (e.g. arguments, other results, etc) to have the first letter in upper case? (Arguments, Other Results, etc)

Sure! I actually realized that inconsistency after sending the pull request ( my bad ), but didn't have time to fix it yet. Thanks for pointing it out!

But, before making the changes, I think it's valuable to wait for more feedbacks as from who you mentioned, so it won't go back and forth on changes.

> border increases visual clutter and is not necessarily a good thing

I disagree, but I respect your opinion and I think it should be take in concern. It's always good to read a counterpoint. In my opinion, a border makes it visually organized and better to distinguish an item from another, especially on quick look, giving the user a better reading experience and the ideia of an 'item area'.

> alpha

Sorry, I don't get this review. Should I change something?

Thanks for the responses and sorry for bad english. : )

File: src/components/DocExplorer.js (hunk 3 above)
Comment (AGS-) [inline src/components/DocExplorer.js]: > Sorry, I don't get this review. Should I change something?
Alphabetical order for the props

**Comment by wincent** [comment]
> > alpha
> Sorry, I don't get this review. Should I change something?

Just a shorthand for "alphabetize these".

**Comment by vinpac** [comment]
> > > alpha
> >
> > Sorry, I don't get this review. Should I change something?
>
> Just a shorthand for "alphabetize these".

I see. Done!

Will this PR me merged?

**Comment by wincent** [comment]
> Will this PR be merged?

Waiting for feedback from somebody with actual design sensibilities. By my own aesthetic judgement, I am not sure this PR is an improvement, but I don't personally feel qualified to review it.

**Comment by asiandrummer** [comment]
@vinpac following up here - I'm also inclined to hear from @leebyron for comments on design improvements. Just wanted to let you know that we haven't forgotten about you and your diff ;)

**Comment by asiandrummer** [comment]
@vinpac following up here again - it's a long overdue PR, let's go ahead and merge this. I'll make some adjustments + rebase to the latest master before merging.

**Comment by kevinsimper** [comment]
I think this looks good 👍 😄

**Comment by vinpac** [comment]
Sure!

**Comment by AGS-** [comment]
@vinpac Awesome thanks!

**Comment by Neitsch** [comment]
@acao  - how do you feel about this PR?

**Comment by acao** [comment]
@Neitsch well, @orta has a whole new design for this, though this would have been very nice to have when it was introduced. I'm not sure we will keep all of it, but perhaps @orta will find it useful to inform the redesign?

**Pull request closed without merging**